google_guest_agent/cfg/cfg.go

```diff
@@ -80,6 +80,10 @@ startup-windows = true
 sysprep-specialize = true
 
 [NetworkInterfaces]
+set_fqdn = true
```

Comment:
We want them on unstable section until it's fully validated. That way we can cut bug fixes/urgent releases without having new (unstable) features enabled.

Reply:
We move them to the proper section whenever doing an actual release of the feature.

Reply:
Do you mean like 
```
[Unstable]
set_fqdn = false
```
Or like 
```
[NetworkInterfaces.Unstable]
set_fqdn = false
```
I'm also making the assumption that these settings should default to off if they are in an unstable state, is that right?

Reply:
Yep, I mean like:

```
[Unstable]
set_fqdn = false
```

and yes, they are defaulted to off.